test/sicp_clojure/exc_1_1.clj

```diff
@@ -0,0 +1,47 @@
+(ns sicp-clojure.exc-1-1
+  (:require [clojure.test :refer :all]
+            [sicp-clojure.core :refer :all]))
+
+(deftest tenIsTen
+  (is (= 10 10)))
+
+(deftest flatAddition
+  (is (= 12 (+ 5 3 4))))
+
+(deftest flatSubtraction
+  (is (= 8 (- 9 1))))
+
+(deftest division
+  (is (= 3 (/ 6 2))))
+
+(deftest mathMix
+  (is (= 6
+    (+ (* 2 4) (- 4 6)))))
+
+(deftest variableAssignmentAndUsage
+  (def a 3)
```

Comment:
It's possible to def in here, but generally it seems discouraged to do this anywhere other than at the top level of the namespace